yarn.lock

```diff
@@ -3487,6 +3487,20 @@
   dependencies:
     defer-to-connect "^2.0.0"
 
+"@testing-library/dom@>=7":
+  version "9.3.4"
+  resolved "https://registry.yarnpkg.com/@testing-library/dom/-/dom-9.3.4.tgz#50696ec28376926fec0a1bf87d9dbac5e27f60ce"
+  integrity sha512-FlS4ZWlp97iiNWig0Muq8p+3rVDjRiYE+YKGbAqXOu9nwJFFOdL00kFpz42M+4huzYi86vAK1sOOfyOG45muIQ==
+  dependencies:
+    "@babel/code-frame" "^7.10.4"
+    "@babel/runtime" "^7.12.5"
+    "@types/aria-query" "^5.0.1"
+    aria-query "5.1.3"
+    chalk "^4.1.0"
+    dom-accessibility-api "^0.5.9"
+    lz-string "^1.5.0"
+    pretty-format "^27.0.2"
+
 "@testing-library/dom@^9.0.0":
   version "9.3.3"
```

Comment:
This results in the wrong `@testing-library/dom` being resolved by yarn which then causes a ton of `act` warnings in tests, see #33356 and [a slack thread](https://gravitational.slack.com/archives/C02DQ1C2BMW/p1703063417879609?thread_ts=1703028787.728229&cid=C02DQ1C2BMW) from the previous time this has happened.

I'll post a PR which fixes that soon.

Reply:
@ravicious Thanks, I wasn't aware of this issue.